datamodel/extra_scripts/oracle/122_idx_gebr_doel.sql

```diff
@@ -0,0 +1,4 @@
+
+CREATE INDEX idx_gebrdoel  ON gebouwd_obj_gebruiksdoel (fk_gbo_sc_identif);
+
+CREATE INDEX mb_kad_onrr_z_ar_overgeg_idx  ON mb_kad_onrrnd_zk_archief (overgegaan_in);
```

Comment:
Deze `mb_kad_onrr_z_ar_overgeg_idx` index in `datamodel/extra_scripts/oracle/207_brk_views.sql opnemen, want de view bestaat niet in de database op het moment dat je deze index wilt aanbrengen...